JeMPI_Apps/JeMPI_UI/src/services/ApiClient.ts

```diff
@@ -31,6 +31,21 @@ import { Config } from 'config'
 import axios from 'axios'
 import { getCookie } from '../utils/misc'
 
+const apiClientAuth = (() => {
```

Comment:
I beleive it was jsut using cookies before. Any reason to change it?

Reply:
Yes, that is correct, however there were issues with setting cookies when the api's domain/ip/node is different to the webapp's domain/ip/node (i.e cross-domain cookie setting ). For this to work we would need to set the `SameSite` property to `None`, which would also require us to set the `Secure` property. Once the `Secure` is set we would then have to use https  for all communications with the api (which means setting up certificates and all that). In addition to this one would still have to set _alllow 3rd party cookies_ in your browser settings for it to work.

Since we are sure of our own api's domain (and the header token is session encrypted) I thought using token headers would be better, than introducing all these overheads. 

As a posterity reference: https://developers.google.com/search/blog/2020/01/get-ready-for-new-samesitenone-secure

Reply:
Cool, sounds logical to me.